packages/ruby/spec/readme/filter_spec.rb

```diff
@@ -1,85 +1,90 @@
-require "readme/filter"
+require 'readme/filter'
 
 RSpec.describe Readme::Filter do
-  Filter = Readme::Filter
-
-  describe ".for" do
-    it "returns RejectParams when only reject argument is given" do
-      result = Filter.for(reject: ["reject"])
-      expect(result).to be_an_instance_of Filter::RejectParams
+  describe '.for' do
+    it 'returns RejectParams when only reject argument is given' do
+      result = described_class.for(reject: ['reject'])
+      expect(result).to be_an_instance_of described_class::RejectParams
     end
 
-    it "returns AllowOnly when only allow_only argument is given" do
-      result = Filter.for(allow_only: ["keep"])
-      expect(result).to be_an_instance_of Filter::AllowOnly
+    it 'returns AllowOnly when only allow_only argument is given' do
+      result = described_class.for(allow_only: ['keep'])
```

Comment:
This was new to me but this `described_class` thing references the class that you set up with `RSpec.describe`. Using this allows us to not have to set `Readme::Filter` to a local `Filter` variable. 💅